lib/angulartics-newrelic-insights.js

```diff
@@ -16,6 +16,7 @@
     .config(['$analyticsProvider', function($analyticsProvider) {
       angulartics.waitForVendorApi('newrelic', 100, function(newrelic) {
         $analyticsProvider.registerEventTrack(function(action, properties) {
+          properties.eventType = 'PageAction';
```

Comment:
I think the solution would be to clone the object before it gets mutated, so something like:

```javascript
$analyticsProvider.registerEventTrack(function(action, properties) {
   var propertiesCopy = angular.copy(properties);
   propertiesCopy.eventType = 'PageAction';
   newrelic.addPageAction(action, properties);
});
```

Also if you could add a link as a comment to some documentation that explains why this is necessary, that would be great.